### DISCUSSION TITLE
Where is new Query Store view in GUI

### DISCUSSION THREAD

**Discussion by jacobgexigo** [trimble-oss/dba-dash#930] [General]
I'm having trouble finding where the new Query Store view is in the GUI for 3.7.0.  Can you tell me where that is?

**Comment by DavidWiseman** [comment]
It's available if you click on an instance or a database in the tree.  Note: For Azure DB you will need to navigate down to the database.  

If you don't see the "Top Queries (Query Store)" tab you might not have messaging enabled.  In the service configuration tool on the options tab, ensure that "Enable Communication" is checked.  The service will need a restart after making this change.

If you have messaging enabled and you still can't see the tab, your user account might not have execute permission on the messaging schema.  There is a Messaging role you can use to grant this.

**Comment by DavidWiseman** [comment]
This highlights where it's available in the tree.  It's also not available on instances older than 2016.

![image](https://github.com/trimble-oss/dba-dash/assets/33157668/30b28f12-c2b0-4e01-86c8-0d91299636f9)

**Comment by jacobgexigo** [comment]
Great, thank you.  I missed the requirement to turn on Communication in order for it to work.

**Reply by DavidWiseman** [reply]
This will also allow you to trigger collections to run on demand and might enable other features down the line.  The query store information is being pulled from the monitored instances rather than from the repository database.
